src/hatti/map/viewby.cljs

```diff
@@ -82,27 +46,29 @@
    eg. a bin for numbers/dates, an option for multiple/single selects."
   [field raw-answers ids]
   (let [fname (:full-name field)
-        ans-s (preprocess-answers field raw-answers)
-        answer->count (frequencies (flatten ans-s))
-        sorted (cond
-                 (or (f/categorical? field)
-                     (f/text? field)
-                     (f/calculate? field))
+        preprocessed-answers (preprocess-answers field raw-answers)
+        answer->count (frequencies (flatten preprocessed-answers))
+        sorted-answers (cond
+                 (or (form-utils/categorical? field)
```

Comment:
Or a pre-commit hook that does auto-indentation. But then we'd have to agree on a rule. Our indentation rules - across the team - seem to differ subtly.